### DISCUSSION TITLE
ZBDongle-E v1 6.10 -> 7.4 upgrade possible?

### DISCUSSION THREAD

**Discussion by aik** [Koenkk/zigbee2mqtt#23824] [Devices]
z2m tells about it" {"meta":{"maintrel":"3 ","majorrel":"6","minorrel":"10","product":8,"revision":"6.10.3.0 build 297"},"type":"EZSP v8"}' 
The upgrade instructions work for ZBDongle-E v2 (which I have lying on my desk) but not for v1 - the bootloader menu does not show up to select "upload firmware".
The ultimate goal is to migrate from v1 to v2 (way too many "c4 error") but googling suggests it is not supported.

So, is the v1 device an e-waste?

**Comment by chris-1243** [comment]
According #23224, v7.4.x firmware should be the maximum you may flash on your dongle (v1).

You might enter the bootloader manually. Insert into the USB port while pressing the bootloader button in order to be able to flash

**Comment by aik** [comment]
long time...
So, I put the dongle in the boot more and managed to flash 7.4.4. So far so good.
A bit worrying thing was though was lots of "Retry 0: NAK on sector" during the fw update:
```
Xmodem sectors/kbytes sent: 1709/213kRetry 0: NAK on sector
Xmodem sectors/kbytes sent: 1712/214kRetry 0: NAK on sector
Retry 0: NAK on sector                                     
Xmodem sectors/kbytes sent: 1713/214kRetry 0: NAK on sector
Retry 0: NAK on sector                                     
Retry 0: NAK on sector                                     
Retry 0: NAK on sector                                     
Retry 0: NAK on sector                                     
```
Should I worry? Is USB extension cable too long (which is usb3 ugreen)? Or device dying?
